ui/app/components/clients/attribution.js

```diff
@@ -114,24 +130,20 @@ export default class Attribution extends Component {
         'Non-entity clients',
       ];
 
-    // each array will be a row in the csv file
-    if (this.isSingleNamespace) {
-      graphData.forEach((mount) => {
-        csvData.push(['', mount.label, mount.clients, mount.entity_clients, mount.non_entity_clients]);
-      });
-      csvData.forEach((d) => (d[0] = this.args.selectedNamespace));
-    } else {
-      graphData.forEach((ns) => {
-        csvData.push([ns.label, '', ns.clients, ns.entity_clients, ns.non_entity_clients]);
-        if (ns.mounts) {
-          ns.mounts.forEach((m) => {
-            csvData.push([ns.label, m.label, m.clients, m.entity_clients, m.non_entity_clients]);
-          });
-        }
-      });
-    }
+    // each array is a row in the csv file
+    graphData.forEach((data) => {
+      let namespace = this.isSingleNamespace ? this.args.selectedNamespace : data;
```

Comment:
I might be wrong about this for this situation, but if this.isSingleNamespace changes, you might want to make it a tracked property. The way I think of it, is in the old non-glimmer computed property style. If namespace here was a computed property and needed to watch if isSingleNamespace changed, then namespace should be a getter and isSingleNamespace should be a tracked property.

Again, this may not be relevant for this code structure, but thought it worth pointing out.

Reply:
Hmm, `this.isSingleNamespace` relies on the `selectedNamespace` arg which _is_ a tracked property in the parent. So the getter should update appropriately based on the actions from the parent, right? So the tracked property in the parent changing, would trigger the getter to update - right?